### ISSUE TITLE
Could not find PARDF in HEMCO data list when running TaggedCO 0.25x0.3125_AS simulation by V14.1.1

### ISSUE THREAD

**Issue by CodeCatLZW** [geoschem/geos-chem#2022]
### Name and Institution (Required)

Name:  Li Zhiwei  
Institution:  University of Science and Technology of China

### Description of your issue or question

I am running  the TaggedCO 0.25x0.3125_AS simulation  from 2023-01-01 to 2023-01-03 by V14.1.1, 
But it stopped, the  GC.log tells  me:
![image](https://github.com/geoschem/geos-chem/assets/75576802/7b48667b-63e0-4da5-84fe-39ca1610b283)

I don't know how to solve this problem.
I used the met_field  is GEOS_0.25x0.3125_AS, I download it by modify   `HEMCO_Config.rc`  likes question  #2021.
My restart file was obtained by simulating 20190701 to 20230101 on geosfp_2x25_TagedCO.
I wonder what's wrong with this, thanks!

**Comment by yantosca** [comment]
Thanks for writing @CodeCatLZW.  Could you attach your configuration files (geoschem_config.yml, HEMCO_Config.rc) and the log files to this issue?  That will help us diagnose the problem.

**Comment by CodeCatLZW** [comment]
Thank you for your response @yantosca . There is my configuration files:
[geoschem_config.yml.log](https://github.com/geoschem/geos-chem/files/13245984/geoschem_config.yml.log)
[HEMCO_Config.rc.log](https://github.com/geoschem/geos-chem/files/13245985/HEMCO_Config.rc.log)
[HEMCO_Config.rc.gmao_metfields.log](https://github.com/geoschem/geos-chem/files/13245988/HEMCO_Config.rc.gmao_metfields.log)
[HISTORY.rc.log](https://github.com/geoschem/geos-chem/files/13245989/HISTORY.rc.log)
[GC.230101-230103-03.log](https://github.com/geoschem/geos-chem/files/13245990/GC.230101-230103-03.log)

Due to the restriction of file types on GitHub, I have added  `.log`  to all of these files (except GC.log), please don't mind it.
If there's anything else I can do, just let me know.

**Comment by CodeCatLZW** [comment]
I still have this problem, if possible, could you help me to look at it, thank you very much. @yantosca

**Comment by CodeCatLZW** [comment]
HI, @yantosca,  I think I've solved the problem.
For the HEMCO_Config.rc.gmao_metfields file, I changed from `LWGNT `to `PARDF`.

before:
![image](https://github.com/geoschem/geos-chem/assets/75576802/a7499457-df3e-479b-b162-a3078327a821)

now:
![image](https://github.com/geoschem/geos-chem/assets/75576802/24f3e9e3-45b1-488b-ba33-013632cbcc7b)

I don't know if I accidentally changed to this field.
Will there be problems with file fields being misaligned during compilation? In fact, the others  simulation files I compiled earlier did not have this problem. This is probably largely my fault.
